authhelper.go

```diff
@@ -0,0 +1,179 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/Pallinder/go-randomdata"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/muir/nfigure"
+	"github.com/pkg/browser"
+)
+
+type Output struct {
```

Comment:
Can this be given a more descriptive name?

Reply:
Changed to `AuthHelperOutput` and added a comment.  Good enough?